### PR TITLE
Gang Huo - Submission of Applying Statistical Concepts' assignment_1.ipynb

### PR DESCRIPTION
**Pull request by ganghuo2024**
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
Completed all the coding tasks and questions in assignment_1.ipynb

## What did you learn from the changes you have made?
Learned how to train, tune and evaluate a KNN model using the relevant Python packages in sklearn, such as KNeighborsClassifier, StandardScaler, train_test_split, GridSearchCV, accuracy_score, etc.  In addition, I learned some important ML concepts such as cross-validation and confusion matrix. 

## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
I think we can also use Logistics Regression or Gradient Boosting to build a similar model. 

## Checklist
- [ ] I can confirm that my changes are working as intended

### CODE REVIEW

**Review by amanda-ng518 — APPROVED** [review]
A1: PASS
